third_party/versions.bzl

```diff
@@ -80,9 +81,9 @@ MAVEN_TEST_DEPENDENCY_VERSIONS = {
     "androidx.test.espresso:espresso-contrib": "3.1.0",
     "androidx.test.espresso:espresso-core": "3.2.0",
     "androidx.test.espresso:espresso-intents": "3.1.0",
-    "androidx.test.ext:junit": "1.1.1",
-    "androidx.test:core": "1.0.0",
-    "androidx.test:runner": "1.2.0",
+    "androidx.test.ext:junit": "1.1.3",
+    "androidx.test:core": "1.4.0",
+    "androidx.test:runner": "1.4.0",
```

Comment:
@BenHenning I've update these version when we were trying the previous implementation.
Shall I revert them?

Reply:
I lean toward not upgrading versions unless specifically needed.

Reply:
Done